libs/spark/src/Unstable_FormLabel/Unstable_FormLabel.tsx

```diff
@@ -30,15 +30,14 @@ const useStyles = makeStyles<Unstable_FormLabelClassKey>(
     root: {
       ...theme.unstable_typography.label,
       color: theme.unstable_palette.text.heading,
-      marginBottom: 8,
       /* focused -- can get from internal context => can't condition on prop */
       '&.Mui-focused': {
         // override Mui default
         color: theme.unstable_palette.text.heading,
       },
       /* error -- can get from internal context => can't condition on prop */
       '&.Mui-error': {
-        color: theme.unstable_palette.red[700],
+        color: theme.unstable_palette.text.heading,
```

Comment:
Why text.heading here?

Reply:
I'm preferring to use the `palette.text` key instead of a `palette.neutral` key where it is the color of text. Whether or not its idiomatic to use the designated heading key for a form label is a different issue, but it is text. And if you're asking why it's red not neutral, that's the decided design.

Reply:
I was more thinking why not red as it is an error state.

Reply:
Ah gotcha. Yeah the rest of the control will be in error styling: input border, value, sub-control labels. Just not the form label.

Reply:
That makes sense then!